config/locales/client.en.yml

```diff
@@ -108,6 +108,16 @@ en:
       ai_persona:
         name: Name
         description: Description
+        no_llm_selected: "No language model selected"
+        max_context_posts: "Max Context Posts"
+        max_context_posts_help: "The maximum number of posts to use as context for the AI when responding to a user. (empty for default)"
```

Comment:
Could be helpful to mention the default is 40, the only problem then is that you have to keep the translation and the server const in sync :D

Reply:
We might be able to pass it down via the `Site` serializer and then use that to pass to the translation/the placeholder here:

![image](https://github.com/discourse/discourse-ai/assets/920448/b17b9201-f21c-4cad-a106-5349cf80ecf7)

Reply:
hmm tricky... yeah maybe this should be a site setting? I can see people who want to save money set it lower ... bot probably operates fine at 10 context and you would save a big pile of money, especially considering giant convos @tgxworld tends to have :)

Reply:
Yeah site setting could be fine as well, at least as the **default**, because then you can use that on the client too for the input and translation.